artemis-server/src/main/java/org/apache/activemq/artemis/core/paging/cursor/impl/PageCounterRebuildManager.java

```diff
@@ -186,6 +189,7 @@ public void run() {
    }
 
    public void rebuild() throws Exception {
+      logger.info("Rebuilding " + pgStore.getStoreName());
```

Comment:
Should be using logger formatting as opposed to string concats.

Reply:
I didn't mean to keep this method.. thank you